modules/gitops-enable-replicas-for-argo-cd-server.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * argo-cd-custom-resource-properties.adoc
+
+:_content-type: PROCEDURE
+[id="enable-replicas-for-argo-cd-server_{context}"]
+= Enable replicas for Argo CD server and Repo server
+
+[role="_abstract"]
+You can enable a number of replicas for the Argo CD-server and Argo CD-repo-server workloads. As these workloads are stateless, the replica count can be increased to distribute the workload better among pods. However, if a horizontal autoscaler is enabled on Argo CD-server, it will be prioritized over the number set for replicas.
+
+.Example
+
+[source,yaml]
+----
+  apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+  labels:
+    example: repo
+spec:
+  repo:
+    replicas: 1
+  server:
+    replicas: 1
+    route:
+      enabled: true
+      path: /
+      tls:
+        insecureEdgeTerminationPolicy: Redirect
+        termination: passthrough
+      wildcardPolicy: None
+----
```

Comment:
- If needed, you can use callouts to explain any user-replaceable values.
- Use `...` to indicate places where the YAML content has been truncated.